ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -340,7 +338,6 @@ public void testRemoveContainerAndUpdateAccount() throws JSONException {
       accountBuilder.build();
     } catch (IllegalStateException e) {
       // expected, as new account id does not match the parentAccountId of the two containers.
-      System.out.println(e.getMessage());
     }
 
     // remove two existing containers.
```

Comment:
in line 343, you are removing all containers. Also, as I suggested, can you remove M no of containers randomly. Also after each removal, check for non existence of the container from the account ref. In the end, you can remove all and do all of your validations.